### ISSUE TITLE
Text Color in TUI Themes

### ISSUE THREAD

**Issue by Ryushin** [quantum/esos#230]
The grey text color is very hard to read on sea green and the bright green dialog backgrounds in the themes.  I'm having to zoom into the window to make out the text.  The grey just does not have enough contrast.  Perhaps change the general text to white and the top menu to a bright yellow with a bright red letter.

**Comment by msmith626** [comment]
Yeah, I've been meaning to add a "high contrast" theme or perhaps simply
black background + white foreground for most/all widgets. Can you produce a
patch for this? I'm having trouble finding time to accomplish this, so if
you're comfortable doing that, it would be greatly appreciated. If not, I
hope to get to it in the near future.

--Marc

**Comment by Ryushin** [comment]
Afraid we both don't have enough time in the day.  I'm still working getting this storage sorted out.  Right now I had to go back to LIO on the native hardware.  I was using ESOS through a VM and passing through the QLogic card.  Though I'm having issues with the transfer dropping to 0 if I use IO Threads on the KVM.  Still working on that.  Still working on this.
